boards/arm64/qemu/qemu-armv8a/configs/fastboot/defconfig

```diff
@@ -16,6 +16,7 @@ CONFIG_ARCH_CHIP_QEMU=y
 CONFIG_ARCH_CHIP_QEMU_A53=y
 CONFIG_ARCH_INTERRUPTSTACK=4096
 CONFIG_AUDIO=y
+CONFIG_BOARD_LOOPSPERMSEC=354375
```

Comment:
qemu use up_delay provided by:
https://github.com/apache/nuttx/blob/master/drivers/timers/arch_alarm.c#L410C20-L410C29
don't need set CONFIG_BOARD_LOOPSPERMSEC.

Reply:
I don't see how the implementations in that file avoid using the coarse delay, unless there is an implementation of the delay functions for QEMU elsewhere?